npm/angular/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@cypress/angular",
   "version": "0.0.0-development",
   "description": "Test Angular Components using Cypress",
+  "private": true,
```

Comment:
Should have been marked as private since we haven't released yet 😅

Reply:
Good catch!